rasa/nlu/tokenizers/tokenizer.py

```diff
@@ -81,21 +87,28 @@ def train(
 
         for example in training_data.training_examples:
             for attribute in MESSAGE_ATTRIBUTES:
-                if example.get(attribute):
-                    if attribute == INTENT:
-                        tokens = self._split_intent(example)
+                if (
+                    example.get(attribute) is not None
+                    and not example.get(attribute) == ""
+                ):
+                    if attribute in [INTENT, ACTION_NAME]:
+                        tokens = self._split_name(example, attribute)
```

Comment:
Minor: Rename to `_split_label`

Reply:
I was trying my best to avoid the word `label` because label is also the Y in DIET and TED...